pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -471,10 +483,11 @@ def create_advisory(self, advisory_type: str, art_advisory_key: str, release_dat
             f"--assigned-to={self.runtime.config['advisory']['assigned_to']}",
             f"--manager={self.runtime.config['advisory']['manager']}",
             f"--package-owner={self.package_owner}",
-            f"--date={release_date}",
         ]
         if batch_id:
             create_cmd.append(f"--batch-id={batch_id}")
+        else:
+            create_cmd.append(f"--date={release_date}")
```

Comment:
what if we want to overwrite the date on batch?

Reply:
That's handled on line 178 https://github.com/openshift-eng/art-tools/blob/55a21647aabd19ae611dc95f13ec8880be6d9a2b/pyartcd/pyartcd/pipelines/prepare_release.py#L178 this command is not used for it